src/main/java/reposense/commits/CommitInfoAnalyzer.java

```diff
@@ -67,7 +68,11 @@ public static CommitResult analyzeCommit(CommitInfo commitInfo, Map<String, Auth
             logger.log(Level.WARNING, "Unable to parse the date from git log result for commit.", pe);
         }
 
-        String message = elements[MESSAGE_INDEX];
+        String message = "";
+        if (elements.length > ELEMENTS_LENGTH_IF_EMPTY_MESSAGE) {
```

Comment:
It seems that you intend to use const `ELEMENTS_LENGTH_IF_EMPTY_MESSAGE` to simplify expression.

Try using a boolean instead.

```
boolean containsMessage =  elements.length > MESSAGE_INDEX;
String message = (containsMessage) ? elements[MESSAGE_INDEX] : "";
```